deploy/deploy_test.go

```diff
@@ -216,7 +300,9 @@ func TestKindSpec(t *testing.T) {
 				"": "local",
 			},
 		},
-		execer:  exec.NewFakeExecer(nil),
+		resp: []fexec.Response{
+			{Cmd: "kind", Args: []string{"create", "cluster", "--name", "test"}, Err: "source container must not be empty"},
```

Comment:
this should not throw an error, this should actually fail from the non-execed code

Reply:
Done